src/app/auth/auth.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams, HttpHeaders } from '@angular/common/http';
+
+import { Observable, of } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+import { User } from './auth';
+
+@Injectable()
+export class AuthService {
+  authUrl = 'https://js-band-store-api.glitch.me/signin';
+
```

Comment:
So here you should have next props that should be available via DI - userToken, userData (name, avatar etc), also you may have here additional isLogging and isLogged props that could be public, and couple of methods (public login(){}, private validateToken() {} - to check whether token is already exists))